renovate.json

```diff
@@ -9,7 +9,12 @@
 		{
 			"packagePatterns": [ "^@wordpress" ],
 			"groupName": "wordpress-monorepo",
-			"separateMajorMinor": false
+			"separateMajorMinor": false,
+			"prPriority": 2
+		},
+		{
+			"packagePatterns": [ "react", "lodash", "^webpack", "typescript" ],
```

Comment:
Does this match any substring? Then I would make `react` more picky, otherwise it will catch every obscure `react-*` package.

I'd also add `babel` to the list.

Reply:
I assume it's a standard regex match, so yeah, substrings would match. Updated it to `^react$` and added babel and redux